inclusive-design/wecount.inclusivedesign.ca
# feat: update the card link on the home page for the "Learn" page (resolves #191)

## Description

DO NOT MERGE: "Learn" page is not yet published, so the link will be broken.

Resolves #191.

## Steps to test

1. Run dev, visit homepage.

**Expected behavior:** Navbar includes link to "Learn" instead of tools, middle card is restored on homepage with text as per #191.

## Additional information

"Learn" page is not yet published, so following the nav link will result in an error. Do not merge until 

## Related issues

- Resolves #191.
